src/adapter/src/catalog/state.rs

```diff
@@ -855,11 +820,15 @@ impl CatalogState {
         return Ok((plan, resolved_ids));
     }
 
+    /// Parses the given SQL string into a pair of [`CatalogItem`].
     pub(crate) fn deserialize_item(&self, create_sql: &str) -> Result<CatalogItem, AdapterError> {
         self.parse_item(create_sql, None, false, None)
     }
 
     /// Parses the given SQL string into a `CatalogItem`.
+    ///
+    /// If `with_enable_for_item_parsing` is true, then a copy of the catalog that has all
```

Comment:
`with_enable_for_item_parsing` is not a param to this function.

Reply:
Fixed.